fullstack-projects/url-shortener/devashmit-day8/backend/main.py

```
@@ -0,0 +1,55 @@
+import database
+
+app = FastAPI(title="URL Shortener API", version="1.0.0")
+
+app.add_middleware(CORSMiddleware,
+    allow_origins=["http://localhost:5173","http://localhost:3000"],
+    allow_methods=["*"], allow_headers=["*"])
+
+@app.on_event("startup")
+def startup(): database.init_db()
+
+class ShortenReq(BaseModel): url: HttpUrl
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)